presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -166,7 +178,7 @@ Mathematical Functions
 .. function:: truncate(x, n) -> double
 
     Returns ``x`` truncated to ``n`` decimal places.
-    ``n`` can be negative to truncate ``n`` digits left of the decimal point. 
+    ``n`` can be negative to truncate ``n`` digits left of the decimal point.
```

Comment:
Remove.

Reply:
Needed to turn "trimTrailingWhitespace" off in the VS code setting.
Done (although this trailing white space should probably be removed regardless).